addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml

```diff
@@ -25,7 +25,11 @@
 			<channel id="extrfanbalancebase" typeId="extrfanbalancebase" />
 			<channel id="suppfanbalancebase" typeId="suppfanbalancebase" />
 			<channel id="homespeedsetting" typeId="homespeedsetting" />
+            <channel id="awayspeedsetting" typeId="awayspeedsetting" />
+            <channel id="boostspeedsetting" typeId="boostspeedsetting" />
 			<channel id="homeairtemptarget" typeId="homeairtemptarget" />
+            <channel id="awayairtemptarget" typeId="awayairtemptarget" />
+            <channel id="boostairtemptarget" typeId="boostairtemptarget" />
```

Comment:
Please reformat this file so tabs are used for indentation.

Reply:
done